### PR TITLE
Support multiple excluded networks

### PR DESCRIPTION
**Pull request by koorgoo**
Related to #155.

### CODE REVIEW

**Comment by koorgoo** [comment]
Noop comment not to close the issue.

**Comment by topas-rec** [comment]
I hope it was fun to create this fix.
It saved me a lot of work - thank you.

I want to report test results for this change.

I created bandwidth limiting for incoming and outgoing connections of an interface.
After that I added five source and five destination networks as exclusions per direction (five excluded networks for incoming and five excluded networks per outgoing)

I verified the result by transferring data
- out of an excluded network into an excluded network and by
- transferring data into and out of a non-excluded network into and out of an excluded network.

The results say that the traffic between excluded networks was not limited, while the traffic to networks that were not excluded was limited.

In short: it works for my use case.

Suggesting to merge this, since it is very helpful for bandwidth management at routers.
